content/en-us/reference/engine/classes/Folder.yaml

```diff
@@ -9,6 +9,13 @@ description: |
   container classes like `Class.Model`, a folder offers no additional
```

Comment:
```suggestion
  container classes like `Class.Model`, a folder generally offers no additional
```

Reply:
Adding "generally" here since UI usage does seem to offer some _slight_ additional functionality versus "nothing but a basic container".